src/xenia/kernel/xam/xam_net.cc

```diff
@@ -288,6 +288,9 @@ dword_result_t NetDll_WSACleanup_entry(dword_t caller) {
 }
 DECLARE_XAM_EXPORT1(NetDll_WSACleanup, kNetworking, kImplemented);
 
+// Instead of using dedicated storage for WSA error like on OS.
```

Comment:
"like on Windows", as POSIX also seems to share error numbers (also this line and the next can be in one sentence).